tests/acceptance/pageObjects/FilesPageElement/filesList.js

```diff
@@ -539,6 +539,12 @@ module.exports = {
         .click(formattedFileRow)
         .waitForElementVisible('@versionsElement')
         .click('@versionsElement')
+    },
+    assertElementNotPresent: function () {
```

Comment:
Implement this function in `phoenixPage` and check for element not present `#Phoenix`.
```suggestion
    assertPageIsEmpty: function () {
```